install/kubernetes/hubble/values.yaml

```diff
@@ -33,6 +33,8 @@ metrics:
   enabled:
    - drop
   address: ":6943"
+  serviceMonitor:
```

Comment:
Would you mind adding a comment that briefly explains what this does?

Reply:
Added